docs/content/preview/explore/ysql-language-features/pg-extensions.md

```diff
@@ -33,6 +33,7 @@ YugabyteDB supports the following [PostgreSQL modules](https://www.postgresql.or
 
 | Module | Description | Examples |
 | :----- | :---------- | :------ |
+| [auto_explain](https://www.postgresql.org/docs/11/auto-explain.html) | Provides a means for logging execution plans of slow statements automatically. | [Example](#auto-explain-example) |
```

Comment:
This link should not refer to PostgreSQL docs since there is stuff  in PG11 which is not applicable. 
Lets break from the precedence and link to the same what is referenced in Example  for this extension. 

example  link can be updated to Reference the actual example, starting from the following line : 

To change auto_explain parameters, use the SET statement. For example: